dotCMS/src/main/java/com/dotcms/publisher/business/PublisherQueueJob.java

```diff
@@ -355,6 +381,46 @@ private void updateBundleStatus(final PublishAuditStatus bundleAudit,
 			}
 		}
 		Logger.info(this, "===========================================================");
+
+		//Growl Notifications
+		if(UtilMethods.isSet(notificationMessage)){
+			final SystemMessageBuilder message = new SystemMessageBuilder()
+					.setMessage(LanguageUtil.format(
+							APILocator.getUserAPI().loadUserById(bundle.getOwner()).getLocale(),
+							notificationMessage,
+							notificationMessageArgument))
+					.setSeverity(notificationSeverity)
+					.setType(MessageType.SIMPLE_MESSAGE)
+					.setLife(notificationLife);
+
+			SystemMessageEventUtil.getInstance().pushMessage(message.create(), ImmutableList.of(bundle.getOwner()));
+		}
+	}
+
+	/**
+	 * Utility method to generate the bundle title for the notification, checks the amount of assets and
+	 * only shows the assetType and assetTitle of the first three items, if there are more the amount of
+	 * remain assets is shown.
+	 *
+	 * @param bundleAssets assets that were added manually to the bundle
+	 * @return a string that will be the argument for the notification
+	 */
+	private String generateBundleTitle(final Map<String,String> bundleAssets)
+			throws LanguageException {
+		int count = 0;
+		String bundleTitle = "";
+		for(final String id : bundleAssets.keySet()){
+			if(count < 3) {
```

Comment:
not sure if this should be configurable in order to get more detail if wanted

Reply:
This is to keep consistency with the Status/History Tab